### ISSUE TITLE
third template called EntityTypeConfiguration.t4

### ISSUE THREAD

**Issue by netbizsystems** [dotnet/EntityFramework.Docs#4088]
Where can I find that third template? I only see the two ["DbContext.t4","EntityType.t4"]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: cae415b7-c0ea-4d96-9d17-7e7d3163c09d
* Version Independent ID: cae415b7-c0ea-4d96-9d17-7e7d3163c09d
* Content: [Custom Reverse Engineering Templates - EF Core](https://learn.microsoft.com/en-us/ef/core/managing-schemas/scaffolding/templates?tabs=dotnet-core-cli)
* Content Source: [entity-framework/core/managing-schemas/scaffolding/templates.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/managing-schemas/scaffolding/templates.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @bricelam
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
@netbizsystems We don't currently have an example of this.

@ErikEJ: @bricelam mentioned that you might be interested in creating a template that uses this.

**Comment by ErikEJ** [comment]
Correct, it is on my TODO list: https://github.com/ErikEJ/EFCorePowerTools/issues/1499

**Comment by R4ND3LL** [comment]
If interested, I created a EntityTypeConfiguration.t4 file located [here](https://github.com/R4ND3LL/EntityFrameworkRuler/blob/main/src/EntityFrameworkRuler.Design/Resources/EntityTypeConfiguration.t4).

This is part of the project [EntityFrameworkRuler.Design](https://www.nuget.org/packages/EntityFrameworkRuler.Design/#readme-body-tab), which provides design time scaffolding services for altering the entity model.    I'm currently using it to make the model match the old EF6 EDMX structure for a few projects I'm upgrading.

Config splitting is an option that can be enabled in there.

**Comment by ErikEJ** [comment]
@R4ND3LL maybe add that tool to the list of tools in the docs?

**Comment by R4ND3LL** [comment]
> @R4ND3LL maybe add that tool to the list of tools in the docs?

Sure.  Where do I go about doing that?

**Comment by ErikEJ** [comment]
Add an entry to the end of this doc and do a PR: https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/extensions/index.md
